src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -176,6 +174,158 @@
+                                <CheckBox x:Uid="PowerLauncher_IncludeInGlobalResult"
+                                          IsChecked="{x:Bind Path=IsGlobal, Mode=TwoWay}"
+                                          IsEnabled="{x:Bind Enabled, Mode=OneWay}"/>
+
+                                <TextBlock x:Uid="PowerLauncher_ActionKeyword"
+                                           x:Name="ActionKeywordHeaderTextBlock"
+                                           Margin="{StaticResource SmallTopMargin}"
+                                           Opacity="{x:Bind DisabledOpacity}"/>
+                                <TextBox x:Uid="PowerLauncher_ActionKeyword"
+                                         Text="{x:Bind Path=ActionKeyword, Mode=TwoWay}"
+                                         Width="86"
+                                         Margin="0,6,0,0"
+                                         AutomationProperties.LabeledBy="{Binding ElementName=ActionKeywordHeaderTextBlock}"
+                                         HorizontalAlignment="Left"
+                                         IsEnabled="{x:Bind Enabled, Mode=OneWay}"/>
+
+                                <TextBlock x:Name="AdditionalOptionsTextBlock"
+                                           x:Uid="Run_AdditionalOptions"
+                                           Margin="{StaticResource SmallTopMargin}"
+                                           Visibility="{x:Bind ShowAdditionalOptions, Converter={StaticResource BoolToVisibilityConverter}}"
+                                           Style="{StaticResource SettingsGroupTitleStyle}"
+                                           Opacity="{x:Bind DisabledOpacity}"/>
+
+                                <ListView ItemsSource="{x:Bind Path=AdditionalOptions}"
+                                          SelectionMode="None"
+                                          IsEnabled="{x:Bind Enabled, Mode=OneWay}">
+                                    <ListView.ItemContainerStyle>
+                                        <Style TargetType="ListViewItem">
+                                            <Setter Property="HorizontalContentAlignment"
+                                                Value="Stretch" />
+                                            <Setter Property="VerticalContentAlignment"
+                                                Value="Stretch" />
+                                            <Setter Property="Padding"
+                                                Value="0,0,0,0" />
+                                        </Style>
+                                    </ListView.ItemContainerStyle>
+                                    <ListView.ItemTemplate>
+                                        <DataTemplate x:DataType="ViewModels:PluginAdditionalOptionViewModel">
+                                            <CheckBox Content="{x:Bind Path=DisplayLabel}" 
```

Comment:
@mykhailopylyp 
Is the lable of the checkbox warped automatically? Otherwise we should enable this.

This is the drive-warning-checkbox with German lable on v0.31.2:
![image](https://user-images.githubusercontent.com/61519853/109187597-5599b780-7792-11eb-879b-f8e6239eecf3.png)

Reply:
Check box breaks labels into lines